internal/provider/resource_ably_app.go

```diff
@@ -128,7 +128,14 @@ func (r resourceApp) Read(ctx context.Context, req tfsdk.ReadResourceRequest, re
 
 	// Fetches all Ably Apps in the account. The function invokes the Client Library Apps() method.
 	// NOTE: Control API & Client Lib do not currently support fetching single app given app id
-	apps, _ := r.p.client.Apps()
+	apps, err := r.p.client.Apps()
+	if err != nil {
+		resp.Diagnostics.AddError(
+			"Error reading Resource",
```

Comment:
Lets specify "Application Resource", instead of just resource - this will be easer to understand for end users.

Reply:
Yeah. As application is already written and using this error format I thought I'd just copy it for now. I'd like the error fixes to be a separate PR if that's okay with you.

Reply:
Sounds good to me

Reply:
Created https://github.com/ably/terraform-provider-ably/blob/main/examples/resources/namespace.tf

Reply:
Created #46